Web Development/BaseX_Calculator/readme.md

```diff
@@ -0,0 +1,20 @@
+# BaseX Calculator
+
+## Description :-
+
+A calculator that can convert numbers from base-X to base-Y. Example: `` Decimal to Binary ``, `` Binary to Octal ``, `` Base 3 to Base 6 ``, ..etc.
+
+- Convert from & to decimal, binary, hexadecimal, octal.
+- User can go beyond than the known bases and convert from & to other bases such as base-15.
+
+## Tech Stack :-
+
+- HTML
+- CSS
+- Javascript
+
+## Run Local
+
+* Open `index.html` in browser.
+* Fill in the corresponding fields.
+* Click Convert to reveal the result.
```

Comment:
follow all the rules which are mentioned in #43 according to that your PR is not created. So make that changes in these file. then I will test it out in my local machine then we will further proceed for merging. mention like this terminologies in Readme.md file and also add Demo Video or gif .
<img width="1080" alt="Screenshot 2022-03-03 at 1 01 42 PM" src="https://user-images.githubusercontent.com/78725162/156784531-0b81d85a-59f2-40af-97d5-c4016923fabb.png">
<img width="1100" alt="Screenshot 2022-03-03 at 1 02 07 PM" src="https://user-images.githubusercontent.com/78725162/156784568-e22e9e7e-7474-4e61-801c-5fc6fb66a2e0.png">